### PR TITLE
Octopus update preprocessing

### PR DESCRIPTION
**Pull request by ChristopheLegendre**
Hi Bryce,
Could you review this PR?
Octopus has  modified the output of its VCF in the version 0.7.4 and up.
This PR take into account the update of the updated fields

This version is not back compatible like you commented out before and I will add that information in the next 0.9.0 release

Let me know if you have comments, questions, suggestions

Thanks,
Best,
Christophe

### CODE REVIEW

File: prep_vcfs_somatic/prep_vcf_somatic_functions.sh [hunk 1]
```diff
@@ -572,7 +572,8 @@ function process_octopus_vcf(){
         VCF=$( decompose ${VCF} )
         VCF=$( make_vcf_upto_specs_for_VcfMerger ${VCF} )
         VCF=$( normalize_vcf ${VCF})
-        VCF=$( rename_fields_in_vcf_header_octopus_specific ${VCF} )
+        # VCF=$( rename_fields_in_vcf_header_octopus_specific ${VCF} )  # We do not need to update the header anymore since the version 0.7.4 of octopus already has the AD and
```
Comment (bryce-turner) [inline prep_vcfs_somatic/prep_vcf_somatic_functions.sh:575]: Might not be a bad idea to add a check for the octopus version, I don't think we want to find out exactly which version the headers changed for, but we could say `if [[ $OCTOPUS_VERSION >= 0.7.4 ]]; then ... fi` <-- pseudo code

If this is non-trivial to find out, disregard this recommendation!

**Review by bryce-turner — APPROVED** [review]
Looks good and is in line with what we've discussed externally. Most of the tracked edits are from indentation changes and other formatting. I noticed that now we don't need to generate as many fields as previously (again like we discussed externally).

Only "negative" comment is that this does not appear to be backwards compatible for octopus. In this regard I would advise updating the vcfmerger versioning to 0.9.0 and be sure to note the change for supported octopus version. More than likely backwards compatibility would really inflate the code base and slow down processing.

**Comment by PedalheadPHX** [comment]
Agree we definately need to do a release that includes this octopus update and MUST be noted in release change log

**Review by bryce-turner — APPROVED** [review]

**Comment by ChristopheLegendre** [comment]
Bug fixed in part making the Header in the `dvmfuncs.py` file.
Sorry about that you may review it again.

**Comment by bryce-turner** [comment]
Merging this but leaving the branch open in case there are any other last minute changes

**Pull request merged**